src/mpid/ch4/shm/ipc/gpu/gpu_post.h

```diff
@@ -12,7 +12,7 @@
     - name        : MPIR_CVAR_CH4_IPC_GPU_P2P_THRESHOLD
       category    : CH4
       type        : int
-      default     : 32768
+      default     : 1024
```

Comment:
In addition, there may be a benefit of having an initial delay threshold. So for a medium message with a new buffer, we use the posix path until a few repeats to indicate it is worthwhile to pay the huge mapping/registration cost.